src/main/java/com/troubleshooting/troubleshootingtool/ContainerDataAccess.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Direct interface with Docker Client using the [Docker Java API Client](https://github.com/docker-java/docker-java)
+ * and extracts environment variables and logs from the defined container
+ */
+package com.troubleshooting.troubleshootingtool;
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.command.ExecCreateCmdResponse;
+import com.github.dockerjava.api.command.LogContainerCmd;
+import com.github.dockerjava.api.model.Frame;
+import com.github.dockerjava.core.DockerClientBuilder;
+import com.github.dockerjava.core.command.ExecStartResultCallback;
+import com.github.dockerjava.core.command.LogContainerResultCallback;
+
+import java.io.ByteArrayOutputStream;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class ContainerDataAccess implements IDataAccess{
+
+    DockerClient dockerClient;
+
+    public ContainerDataAccess() {
+        dockerClient = DockerClientBuilder.getInstance().build();
+    }
+
+    public String getEnv(String containerID) {
```

Comment:
method name is unclear, what does it do?

Reply:
is it okay to call the method `getEnvironmentInfo` if I have the same method name in another class?

Reply:
only if makes sense in the context of your code, and it is not a 'smell'